backend/Tournament/serializers.py

```diff
@@ -0,0 +1,128 @@
+from rest_framework import serializers
+from rest_framework.utils.serializer_helpers import ReturnDict
+
+from .models import Tournament, \
+						TournamentPlayer
+from Match.models import Match
+from Match.matchState import MatchState
+
+class TournamentSerializer(serializers.ModelSerializer):
+	state_display = serializers.CharField(source='get_state_display', read_only=True)
+
+	class Meta:
+		model = Tournament
+		fields = ['id', 'state_display']
+
+class TournamentPlayerSerializer(serializers.ModelSerializer):
+	class Meta:
+		model = TournamentPlayer
+		fields = ['id', 'user', 'display_name']
+
+class TournamentMatchSerializer(serializers.ModelSerializer):
+	first_tournament_player = serializers.SerializerMethodField()
+	second_tournament_player = serializers.SerializerMethodField()
+	winner = serializers.SerializerMethodField()
+	class Meta:
+		model = Match
+		fields = ['id', 'tournament_match_id', 'first_tournament_player', 'second_tournament_player', 'winner']
+
+	def get_first_tournament_player(self, match: Match) -> TournamentPlayerSerializer | None:
```

Comment:
first and second should be changes to left and right as its the way in other parts of this software!